schema/migration-2-0029-20231017.sql

```diff
@@ -0,0 +1,29 @@
+-- Persistent generated migration.
+
+CREATE FUNCTION migrate() RETURNS void AS $$
+DECLARE
+  next_version int ;
+BEGIN
+  SELECT stage_two + 1 INTO next_version FROM schema_version ;
+  IF next_version = 29 THEN
+    EXECUTE 'ALTER TABLE IF EXISTS "pool_offline_data" RENAME TO "off_chain_pool_data"' ;
+    EXECUTE 'ALTER TABLE IF EXISTS "off_chain_pool_data" RENAME CONSTRAINT "unique_pool_offline_data" TO "unique_off_chain_pool_data"' ;
+
+    EXECUTE 'ALTER TABLE IF EXISTS "pool_offline_fetch_error" RENAME TO "off_chain_pool_fetch_error"' ;
+    EXECUTE 'ALTER TABLE IF EXISTS "pool_offline_fetch_error" RENAME CONSTRAINT "unique_pool_offline_fetch_error" TO "unique_off_chain_pool_fetch_error"' ;
```

Comment:
I think this will create the constraint if the old one exists.However when starting from a fresh db, the constraint won't ever be created, due to the `IF EXISTS`